variables.tf

```diff
@@ -125,15 +141,32 @@ variable "vcenter_user_name" {
 }
 
 variable "s3_url" {
+  default = "https://s3.example.com"
```

Comment:
An S3 URL default does not appear to be needed.

This is a broken default, perhaps this should used in the description as an example, with a default of `""` or no default at all.  

The Anthos module can set this value to whatever it wants.

Reply:
Same comment applied to the s3_bucket_name, s3_access_key, and s3_secret_key variables.

Reply:
These all need defaults now since I've added the option to use Google Cloud Storage instead of S3. We need to have defaults or terraform will require these variables to be set.

Reply:
If we require https://github.com/equinix/terraform-metal-vsphere/pull/7#discussion_r568917628 (no default for `objectstorage_tool`, with a description stating  `s3` and  `gcs` are valid choices) we could make these tool-specific choices default to `""`, making them optional.

Reply:
We have `s3_boolean` for this.
If it's true. You use S3 (The default) if false. You use GCS.

Reply:
This has been completely changed now...